openpype/hosts/max/plugins/load/load_camera_fbx.py

```diff
@@ -36,14 +39,26 @@ def load(self, context, name=None, namespace=None, data=None):
         container_name = f"{name}_CON"
 
         asset = rt.getNodeByName(f"{name}")
-        # rename the container with "_CON"
-        container = rt.container(name=container_name)
-        asset.Parent = container
 
-        return container
+        return containerise(
+            name, [asset], context, loader=self.__class__.__name__)
+
+    def update(self, container, representation):
+        from pymxs import runtime as rt
+
+        path = get_representation_path(representation)
+        node = rt.getNodeByName(container["instance_node"])
+
+        alembic_objects = self.get_container_children(node, "AlembicObject")
+        for alembic_object in alembic_objects:
+            alembic_object.source = path
```

Comment:
well, I am not sure about those alembic thingies in th FBX loader? The same is in the max scene loader..

Reply:
umm..,my bad, forgot to rename those variables.